src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/ResolveDataLocks/AcceptDataLocksRequestChangesCommandHandler.cs

```diff
@@ -60,11 +60,11 @@ protected override async Task Handle(AcceptDataLocksRequestChangesCommand reques
                 var dataLockWithUpdatedTraining = dataLocksToBeAccepted.FirstOrDefault(m => m.IlrTrainingCourseCode != apprenticeship.CourseCode);
                 if (dataLockWithUpdatedTraining != null)
                 {
-                    var training = await _trainingProgrammeLookup.GetTrainingProgramme(dataLockWithUpdatedTraining.IlrTrainingCourseCode);
+                    var training = await _trainingProgrammeLookup.GetCalculatedTrainingProgrammeVersion(dataLockWithUpdatedTraining.IlrTrainingCourseCode, dataLockWithUpdatedTraining.IlrEffectiveFromDate.GetValueOrDefault());
```

Comment:
Interesting we use ILrEffectiveFromDate. I was assuming we would be using the Apprenticeship StartDate (which can't be blank). 

I didn't think about the ILR Accept update including the Start date. But I can't see where we update the Start Date in this area of code (might happen else where).

Reply:
Good shout there Paul, have used apprenticeship.StartDate.GetValueOrDefault() in place actually, will run this up against my local test data.